src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php

```diff
@@ -40,6 +47,14 @@ public function __construct(LoggerInterface $logger = null)
     }
 
     /**
+     * @internal
+     */
+    public function setArgumentResolverManager(ArgumentResolverManager $argumentResolverManager)
```

Comment:
Btw, adding a public method is not considered a BC break according to the promise:

 > Add public method	Yes	Yes